### PR TITLE
Feat: Mobile Pincode manager

### PR DESCRIPTION
**Pull request by amadeu2**
# Description of change

Please write a summary of your changes and why you made them. 

## Links to any relevant issues
Closes https://github.com/iotaledger/firefly/issues/2105

Be sure to reference any related issues by adding `fixes issue #`.

## Type of change

Choose a type of change, and delete any options that are not relevant.

- Breaking (any change that would cause existing functionality to not work as expected)
- New (a change which implements a new feature)
- Update (a change which updates existing functionality)
- Fix (a change which fixes an issue)
- Docs (changes to the documentation)
- Chore (refactoring, build scripts or anything else that isn't user-facing)

## How the change has been tested

Describe the tests that you ran to verify your changes.

Make sure to provide instructions for the maintainer as well as any relevant configurations.

## Change checklist

Add an `x` to the boxes that are relevant to your changes, and delete any items that are not.

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that my latest changes pass CI tests
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by nicole-obrien — CHANGES_REQUESTED** [review]
How come we are making a making a function that returns an object of three functions and assigning the result of calling that function to a variable, instead of the original skeleton which was an object with three functions?

**Comment by amadeu2** [comment]
> How come we are making a making a function that returns an object of three functions and assigning the result of calling that function to a variable, instead of the original skeleton which was an object with three functions?

I tend to use this pattern with Svelte for state encapsulation but here is no sense, like you said it's better to use just an object literal to simplify. Just fixed that, thanks.

File: packages/mobile/capacitor/lib/pincodeManager.ts [hunk 1]
```diff
@@ -1,45 +1,46 @@
...
-     * Verifies user entered pincode against the one stored in keychain
-     *
-     * @method verify
-     *
-     * @param {string} key
-     * @param {string} pincode
-     *
-     * @returns {Promise}
+     * Gets pincode from keychain
+     * @method get
```
Comment (nicole-obrien) [inline packages/mobile/capacitor/lib/pincodeManager.ts:24]: Should we keep the original comment?

Comment (amadeu2) [reply packages/mobile/capacitor/lib/pincodeManager.ts]: fixed, thanks!

**Review by nicole-obrien — APPROVED** [review]
LGTM 👩‍🎤

**Pull request merged**